app/views/layouts/application.html.erb

```diff
@@ -23,7 +23,11 @@
         <div class="collapse navbar-collapse" id="navbarsExampleDefault">
           <ul class="navbar-nav mr-auto">
             <li class="nav-item active">
+            <% if signed_in? %>
```

Comment:
Ed's changes remove "documents" entirely if not signed in, which i think looks a lot better. Would you mind removing this?